docs/current_docs/quickstart/package.mdx

```diff
@@ -0,0 +1,171 @@
+---
+slug: /quickstart/package
+hide_table_of_contents: true
+title: "Daggerize the application's packaging"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Quickstart
+
+## Daggerize the application's packaging
+
+At this point, you've written Dagger Functions to test, build and deliver the application to your local host. But why not also package the application and publish a containerized image to a registry?
+
+<Tabs groupId="language">
+<TabItem value="Go">
+
+Update the `dagger/main.go` file and create a new Dagger Function to copy the build output directory into an NGINX web server container image.
+
+```go file=./snippets/package/go/main.go
```

Comment:
This would be a good opportunity to reuse the `build()` function and show how that works. So the input argument would be `source`, and we call `m.Build(source)/self.build(source)/this.build(source)`

Reply:
Done